tests/integration/yamls/receiver-mock.yaml

```diff
@@ -26,16 +26,18 @@ spec:
         - ports:
             - containerPort: 3000
             - containerPort: 3001
-          image: sumologic/kubernetes-tools:2.12.0
```

Comment:
This can't be merged until a new version of k8s-tools is released. Currently it is still undergoing some changes.